### PR TITLE
Add inactive status and apply it to filters

### PR DESCRIPTION
**Pull request by kennymkchan**
Added: 
- Status attribute to property objects
- Change status to inactive after auto win completes
- Filter all to not show inactive properties (apply filter initially and on all subsequent filters)

### CODE REVIEW

**Comment by kennymkchan** [comment]
@KennyKYLam Take a look at this please. Not sure if I missed a place in your filtering logic (:

**Comment by kennymkchan** [comment]
Took a while to find, but please have another look!

File: api/views.py [hunk 1]
```diff
@@ -60,15 +65,15 @@ def post(self, request):
             orQuery = [Q(x) for x in orPredicates]
 
             if not andQuery and not orQuery:
-                listings = Property.objects.all()
+                listings = Property.objects.filter().exclude(status="inactive")
```
Comment (KennyKYLam) [inline api/views.py]: .all() might be more clear?

**Review by KennyKYLam — APPROVED** [review]
Looks good 👍

File: api/views.py (hunk 1 above)
Comment (kennymkchan) [reply api/views.py]: addressed @KennyKYLam

**Pull request merged**